match2/wikis/starcraft2/match_group_legacy.lua

```diff
@@ -7,13 +7,26 @@ local getDefaultMapping = require("Module:MatchGroup/Legacy/Default").get
 local Logic = require("Module:Logic")
 local Lua = require("Module:Lua")
 local Table = require("Module:Table")
+local Variables = require("Module:Variables")
 local String = require("Module:StringUtils")
 
+local _IS_USERSPACE = false
+local _USERSPACE_NUMBER = 2
```

Comment:
It's a little pedantic, but 
```suggestion
local _NAMESPACE_USER = 2
```

Is better in case we add more namespace constants. A module would also be a good alternative for the future

Reply:
then i would rather use `_NAMESPACE_USER_NUMBER` (in case we also have to use the names somewhere)

Reply:
I dislike specifying the type in a variable name so then I would go for `_NAMESPACE_ID_USER`

Reply:
went with `_NAMESPACE_USER`